bdd-generator/src/main/java/com/epam/test_generator/entities/StepSuggestion.java

```diff
@@ -111,10 +114,13 @@ public void setSteps(List<Step> steps) {
 
     public void add(Step step) {
         steps.add(step);
+        this.lastUsedDate = ZonedDateTime.now();
+
     }
 
     public void remove(Step step) {
         steps.remove(step);
+        this.lastUsedDate = ZonedDateTime.now();
```

Comment:
Clarify with Daria

Reply:
Clarified. She said that this isn't needed.